ui/guest/src/react/ZoneMenu.tsx

```diff
@@ -135,9 +135,13 @@ export function ZoneMenu(props: ZoneMenuProps) {
     // endregion
     [modelId, recordType, iceRecord]
   );
-
-  const isItemFile = collection ? Boolean(collection[elementIndex]?.hasOwnProperty('key')) : false;
-  const collectionContainsFiles = collection ? collection.some((item) => item.hasOwnProperty('key')) : false;
+  // File validations only applies to node-selector, not to repeating-group
```

Comment:
While testing, I saw that duplicate option was not showing in a repeat-group with taxonomies, since they have a 'key' property. In general, this `isItemFile` and `collectionContainsFiles` validation was done to handle files inside node-selector.